core/src/test_util/dummy_vdaf.rs

```diff
@@ -4,20 +4,24 @@ use prio::{
     codec::{CodecError, Decode, Encode},
     vdaf::{self, Aggregatable, PrepareTransition, VdafError},
 };
+use rand::random;
 use std::fmt::Debug;
 use std::io::Cursor;
 use std::sync::Arc;
 
 type ArcPrepInitFn =
     Arc<dyn Fn(&AggregationParam) -> Result<(), VdafError> + 'static + Send + Sync>;
-type ArcPrepStepFn =
-    Arc<dyn Fn() -> Result<PrepareTransition<Vdaf, 0, 16>, VdafError> + 'static + Send + Sync>;
+type ArcPrepStepFn = Arc<
```

Comment:
I do think that Janus should use libprio's dummy VDAF. However, I opted not to make that change in this PR because this change is already quite large. 

In any case, when I revisited this file, I found that we can actually revert the change to this file and remove a bunch of noise from elsewhere in the diff, so I did just that.